SETUP/tests/README.md

```diff
@@ -4,13 +4,7 @@ These are phpunit tests.
 
 ## Version requirements
 
-The phpunit tests, as-coded, will work with phpunit 5.7, 6.x, and 7.x. They
-will not work in 8.x or later because the setUp() and tearDown() functions
-do not specify void return types. (See the
-[phpunit8 release notes](https://phpunit.de/announcements/phpunit-8.html).)
-That capability isn't available until PHP 7.1, which the DP code will
-nominally work with, but isn't available on the TEST server or in many
-modern distros.
+The phpunit tests, as-coded, will work with phpunit 6.x, 7.x, 8.x, and 9.x.
```

Comment:
These will still work with phpunit 6 and 7? I assumed they would only work with 8 and later.

Reply:
well for Php 7.4 on my machine at least:
phpunit 6:
![image](https://user-images.githubusercontent.com/1526781/98318049-61732b00-1fa3-11eb-8b98-92606f007c8f.png)

phpunit 7:
![image](https://user-images.githubusercontent.com/1526781/98318082-70f27400-1fa3-11eb-99ba-eefadb481121.png)

ignore the failure, I think it is a db configuration issue.